src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -129,7 +128,8 @@ osx_output_test_default_device(void)
 }
 
 OSXOutput::OSXOutput(const ConfigBlock &block)
-	:AudioOutput(FLAG_ENABLE_DISABLE|FLAG_PAUSE)
+	:AudioOutput(FLAG_ENABLE_DISABLE|FLAG_PAUSE),
+	 ring_buffer(nullptr)
```

Comment:
This doesn't need to be initialized here. If you do anyway with an arbitrary value like `nullptr`, you lose the chance to detect uninitialized variables with tools like valgrind.

Reply:
Done in https://github.com/MusicPlayerDaemon/MPD/pull/598